maven-wrapper/pom.xml

```diff
@@ -33,27 +33,43 @@ under the License.
   <name>Maven Wrapper Jar</name>
   <description>Maven Wrapper Jar download, installs and launches installed target Maven distribution as part of Maven Wrapper scripts run.</description>
 
+  <properties>
+    <junit.version>5.9.1</junit.version>
+    <mockito.version>4.11.0</mockito.version>
+  </properties>
+
   <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
       <version>2.2</version>
```

Comment:
No property here as hamcrest was last released in 2019, and is unlikely to change in the future. Might want to swap Hamcrest for AssertJ, as it's only in two test classes. Suggest to tackle that separately from this one.

Reply:
There's [no automated migration for Hamcrest yet](https://github.com/openrewrite/rewrite-testing-frameworks/issues/212), but that could be added in the future.